resources/views/consumptions/edit.blade.php

```diff
@@ -0,0 +1,50 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="bar-container">
+    <div class="bar-full">
+        <div class="bar-header">
+            <p class="bar-title">Eingang bearbeiten</p>
+        </div>
+    </div>
+    <div class="panel">
+
+        <div class="mininav margin-8">
+            <a href="javascript:history.back()">
+                <img class="link-item" src="https://image.flaticon.com/icons/png/512/318/318276.png"></a>
+        </div>
+
+        <table class="table table-large">
+
+            <tr>
+                <th class="table-header">Datum</th>
+                <th class="table-header text-align-right">Menge</th>
+            </tr>
+
+            <tr>
+                <td class="col">{{$consumption->created_at}}</td>
+                <td class="col col2">{{ $consumption->amount_consumption }}</td>
+            </tr>
+
+        </table>
+
+        <form action="/consumptions/{{ $consumption->id }}" class="mininav" method="post" enctype="multipart/form-data">
```

Comment:
enctype="..." kann weg wenn in diesem formular keine dateien hochgeladen werden.